src/modules/experts/store/expertsSlice.ts

```diff
@@ -68,7 +72,14 @@ const initialState: IExpertsState = {
       loading: LoadingStatusEnum.idle,
       error: null,
     },
-    filters: [],
+    filters: {
+      [FilterTypeEnum.DIRECTIONS]: {
+        value: [],
+      },
+      [FilterTypeEnum.REGIONS]: {
+        value: [],
```

Comment:
```suggestion
        value: {},
      },
      [FilterTypeEnum.REGIONS]: {
        value: {},
```

Reply:
seems like we are using an array of string values as a `value`, isn't it?

Reply:
in `FilterForm` we assign an initial checkbox state of type `{ [key: string]: boolean; }` and later call `setExpertsRegionsFilter` with that value